server/utils.go

```diff
@@ -279,3 +280,24 @@ func getUlimitsFromConfig(config Config) ([]ulimit, error) {
 func translateLabelsToDescription(labels map[string]string) string {
 	return fmt.Sprintf("%s/%s/%s", labels[types.KubernetesPodNamespaceLabel], labels[types.KubernetesPodNameLabel], labels[types.KubernetesContainerNameLabel])
 }
+
+// Validate given hostIP IP belongs to the current host
+// adapted from github.com/kubernetes/kubernetes/pkg/kubelet/kubelet_node_status.go
+func validateHostIP(hostIP net.IP) error {
+	if hostIP.To4() == nil {
```

Comment:
I think IPv6 will be allowed with 1.16.0 too, just to consider for the forward-port.

Reply:
Okay, I'll add it in the forward port :)